Add the privacy policy for the Opf DS Community Meetup (#39)
aicoe-aiops/operate-first-data-science-community: docs/privacy-policy.md

# Privacy Policy

#### _Effective:_ February 8, 2022

This policy document explains how the Operate First Data Science Community Meetup (“we” / “us” / “our”) collects, stores, and uses attendees’ personal information. If you wish to access any particular section of this Privacy Policy, use the list below to jump to that section.

1. [Collection Of Information](#Collection-Of-Information)

2. [Use Of Information](#Use-Of-Information)

3. [Sharing Of Information](#Sharing-Of-Information)

4. [Additional Details](#Additional-Details)

5. [References](#References)

6. [Contact Us](#Contact-Us)

## Collection Of Information

We collect information about you when you:

1. Login using a google account;
2. Use the google meet platform;
3. Attend an event;
4. Visit our website; and

> 5. Communicate with us.

Reviewer: Shall we add the means of communication? is it emails, google chat, slack or more?

Reviewer 2: Yes, "communicate with us" is vague. What does it mean?

Reviewer 3: The list structure here can be confusing. We don't collect information when someone logs in to their gmail or uses the google meet platform. We only collect information when they attend the DS meetup event. 
So maybe just three points
1) Attend DS community meetup through google meet platform
2) Visit our website
3) Communicate using ....
WDYT?

Author: Sounds good, we have rephrased this section :+1:

> We may also collect information, such as usage statistics, by using cookies, server logs, and similar technologies.

Reviewer 3: Should we leave this out until we actually use and track these statistics? And update the doc with exact stats that we collect once we do?

## Use Of Information

We use information about you to:

> 1. Operate and promote our Community;

Reviewer 2: What Does "operate" mean here? We use it to promote upcoming events, but I'm not sure about "operate"

2. Track engagement;
3. Communicate with you; and
4. Improve content.

## Sharing Of Information

We do not share information about you except as described in this Privacy Policy. We may disclose personal information about you, including:

1. To authorities to comply with legal obligations or requests and to protect and defend our rights and property;

> 2. To visualize key performance indicator metrics such as attendees over time, distribution of organizations represented in audience, attendees vs topic presented, average time duration of the meeting, etc. For calculating such KPIs, we anonymized user data, and store the user to anonymous id (e.g. SHA) mapping somewhere secure where very limited people have access.

Reviewer 2: ```suggestion
2. To visualize key performance indicator metrics such as attendees over time, distribution of organizations represented in audience, attendees vs topic presented, average time duration of the meeting, etc. For calculating such KPIs, we anonymized user data, and store the anonymous id (e.g. SHA) mapping somewhere secure where very limited people have access.
```

Why do we need to keep the SHA mapping?

# Privacy Policy

#### _Effective:_ February 8, 2022

This policy document explains how the Operate First Data Science Community Meetup (“we” / “us” / “our”) collects, stores, and uses attendees’ personal information. If you wish to access any particular section of this Privacy Policy, use the list below to jump to that section.

1. [Collection Of Information](#Collection-Of-Information)

2. [Use Of Information](#Use-Of-Information)

3. [Sharing Of Information](#Sharing-Of-Information)

4. [Additional Details](#Additional-Details)

5. [References](#References)

6. [Contact Us](#Contact-Us)

## Collection Of Information

We use Google Meet to host the meetups. Through Google Meet, we get the following information about you, when you attend the Operate First Data Science Community Meetups:
1. Your name

    *a.* as it appears on your Google Account, in case you join through your Google Account; or

    *b.* as you provide in the “Your name” section, in case you join anonymously

2. Email ID or phone number you used to join the event
3. Timestamp(s) when you joined the event
4. Timestamp(s) when you left the event

> We use **Google Analytics** on the operate-first website. This means that, through Google Analytics, we get information such a your Device's Internet Protocol address (e.g. IP address), browser type, browser version, the pages of our Service that You visit, the time and date of Your visit, the time spent on those pages, unique device identifiers and other diagnostic data. We do **not** get any personal information through google analytics.

Reviewer: ```suggestion
We use **Google Analytics** on the operate-first website. This means that when you visit [operate-first](https://www.operate-first.cloud/) website, through Google Analytics, we get information such as your Device's Internet Protocol address (e.g. IP address), browser type, browser version, the pages of our Service that you visit, the time and date of your visit, the time spent on those pages, unique device identifiers and other diagnostic data. We do **not** get any personal information through google analytics.
```

```
 ..and other diagnostic data. We do **not** get any personal information through google analytics.
```

 The term `other diagnostic data` is making me think what other metrics could be there that is shared? Can we be more specific? wdyt? 

@aakankshaduggal , wonderful doc :clap: . I just added some comments and suggestions. Let me know if it makes sense to you. :) Thanks

# Privacy Policy

#### _Effective:_ February 8, 2022

This policy document explains how the Operate First Data Science Community Meetup (“we” / “us” / “our”) collects, stores, and uses attendees’ personal information. If you wish to access any particular section of this Privacy Policy, use the list below to jump to that section.

1. [Collection Of Information](#Collection-Of-Information)

2. [Use Of Information](#Use-Of-Information)

3. [Sharing Of Information](#Sharing-Of-Information)

4. [Additional Details](#Additional-Details)

5. [References](#References)

6. [Contact Us](#Contact-Us)

## Collection Of Information

> We use Google Meet to host the meetups. Through Google Meet, we get the following information about you, when you attend when you attend the Operate First Data Science Community Meetups:

Reviewer: Just a small typo:
```suggestion
We use Google Meet to host the meetups. Through Google Meet, we get the following information about you, when you attend the Operate First Data Science Community Meetups:
```

1. Your name

    *a.* as it appears on your Google Account, in case you join through your Google Account; or

    *b.* as you provide in the “Your name” section, in case you join anonymously

2. Email ID or phone number you used to join the event
3. Timestamp(s) when you joined the event
4. Timestamp(s) when you left the event

We use Google Analytics on the operate-first website. This means that, through Google Analytics, we get the following information about you, when you visit our website:

> Where you got to the page from (e.g. mail,github, twitter, youtube, etc.)
> We do **not** get any personal information through this.

Reviewer: I guess these two points should be like a bullet points right? If so, please reformat the text. They are showing up in a single line. 
```suggestion
Where you got to the page from (e.g. mail,github, twitter, youtube, etc.)
We do **not** get any personal information through this.
```

We may also collect information about you such as your name and email ID, when you communicate with us via email or slack.

In addition to this, we also collect visitor metrics and related information like where did you get to know about this page, whenever you visit the website.

## Use Of Information

We use information about you to:

1. Promote our Community;
2. Track engagement;
3. Communicate with you; and
4. Improve content.

## Sharing Of Information

> We do not share information about you except as described in this Privacy Policy. We may disclose personal information about you, including:

Reviewer: Just a thought on making it a bit concise. Lmk if it makes sense.
```suggestion
We do not share information about you except as described in this Privacy Policy below,
```

Author: I think it's vital to mention that we might be using this personal information for various reasons as mentioned here.

# Privacy Policy

#### _Effective:_ February 8, 2022

This policy document explains how the Operate First Data Science Community Meetup (“we” / “us” / “our”) collects, stores, and uses attendees’ personal information. If you wish to access any particular section of this Privacy Policy, use the list below to jump to that section.

1. [Collection Of Information](#Collection-Of-Information)

2. [Use Of Information](#Use-Of-Information)

3. [Sharing Of Information](#Sharing-Of-Information)

4. [Additional Details](#Additional-Details)

5. [References](#References)

6. [Contact Us](#Contact-Us)

## Collection Of Information

We use Google Meet to host the meetups. Through Google Meet, we get the following information about you, when you attend the Operate First Data Science Community Meetups:
1. Your name

    *a.* as it appears on your Google Account, in case you join through your Google Account; or

    *b.* as you provide in the “Your name” section, in case you join anonymously

2. Email ID or phone number you used to join the event
3. Timestamp(s) when you joined the event
4. Timestamp(s) when you left the event

> We use Google Analytics on the operate-first website. This means that, through Google Analytics, we get the following information about you, when you visit our website:

Reviewer 3: Google analytics collects a lot of information such as location of the user, number of times they logged in, which page they spent the most time at etc., PTAL at a [google analytics privacy policy template.](https://www.termsfeed.com/public/uploads/2022/01/sample-google-analytics-privacy-policy-template.pdf) maybe we can borrow something from here.

Author: Thanks Shrey, will look into this :+1:

> Where you got to the page from (e.g. mail,github, twitter, youtube, etc.)

Reviewer 2: Do we actually get individual user info here, like email address? Or just "some unique user" went to N pages?

Reviewer 4: +1

Author: It's mainly just the number of users, let me rephrase this to make it easier to understand.

We do **not** get any personal information through this.

We may also collect information about you such as your name and email ID, when you communicate with us via email or slack.

> In addition to this, we also collect visitor metrics and related information like where did you get to know about this page, whenever you visit the website.

Reviewer 2: This sounds like a repeat of "we get the following information about you, when you visit our website:" above. How is it different?

Author: Will remove this.

> 

Reviewer 2: We repeatedly say "we get information about *YOU*" but then note, "we do not get any personal information".  This is a little confusing to me, What actual identifying information do we get about users through google analytics?

Reviewer 4: +1
